MobileTraining/MobileTraining/Models/Product.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Product.swift
+//  MobileTraining
+//
+//  Created by Svetoslav Donkov on 24.05.23.
+//
+
+import Foundation
+
+struct Product: Codable, Identifiable {
+    let id: Int
+    let title: String
+    let description: String
+    let short_description: String
```

Comment:
`short_description` is an invalid syntax for Swift. Name it `shortDescription` and write CodingKeys to de/serialize it into `short_description` as showcased in one of the lectures (lecture 3, I think?).